lib/clients/index.js

```diff
@@ -0,0 +1,57 @@
+function MetricClients(backends) {
+  this.clients = [];
+  if (backends) {
+    this.init(backends);
+  }
+}
+
+/**
+ * Init clients for each of the backend specified in the given backends
+ * 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)